libs/application/templates/estate/src/fields/EstateMembersRepeater/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { FC, useEffect } from 'react'
+import { useFieldArray, useFormContext } from 'react-hook-form'
+import { useLocale } from '@island.is/localization'
+import { FieldBaseProps } from '@island.is/application/types'
+import {
+  Box,
+  GridColumn,
+  GridRow,
+  Button,
+  ProfileCard,
+} from '@island.is/island-ui/core'
+import { format as formatNationalId } from 'kennitala'
+import { m } from '../../lib/messages'
+import { EstateRegistrant } from '@island.is/clients/syslumenn'
+import { Answers, EstateMember } from '../../types'
+import { AdditionalEstateMember } from './AdditionalEstateMember'
+
+export const EstateMembersRepeater: FC<FieldBaseProps<Answers>> = ({
+  application,
+  field,
+  errors,
+}) => {
+  const { id } = field
+  const { formatMessage } = useLocale()
+  const { fields, append, remove } = useFieldArray<EstateMember>({
+    name: id,
+  })
+  const { setValue } = useFormContext()
+
+  const externalData = application.externalData.syslumennOnEntry?.data as {
+    relationOptions: string[]
+    estate: EstateRegistrant
+  }
+  const relations =
+    externalData.relationOptions?.map((relation) => ({
+      value: relation,
+      label: relation,
+    })) || []
+  const error = (errors as any)?.estate?.estateMembers
+
+  const handleAddMember = () =>
+    append({
+      nationalId: '',
+      initial: false,
+      enabled: true,
+      name: '',
+    })
+
+  useEffect(() => {
+    if (fields.length === 0 && externalData.estate.estateMembers) {
+      append(externalData.estate.estateMembers)
+    }
+  }, [])
+
+  return (
+    <Box marginTop={2} marginBottom={5}>
+      <GridRow>
+        {fields.reduce((acc, member, index) => {
+          if (member.nationalId === application.applicant) {
+            if (
+              application.answers?.applicantRelation &&
+              application.answers.applicantRelation !== member.relation
+            ) {
+              member.relation = application.answers.applicantRelation
```

Comment:
For all these application.answers.something should we be using getValueViaPath instead?
Just thinking it might be a bit more robust but still it doesnt let you know if the typing of the application changes whereas this approach does. Just thinking out loud here, feel free to chime in.